SimPEG/Survey.py

```diff
@@ -205,6 +205,7 @@ class BaseSurvey(object):
     __metaclass__ = Utils.SimPEGMetaClass
 
     std = None       #: Estimated Standard Deviations
+    eps = None       #: Estimated Noise Floor
```

Comment:
I suppose these would be better in the data class? But here is a good place for them now.